package.json

```diff
@@ -40,7 +40,7 @@
     "clean:obsolete-snapshots": "npm test -- -u",
     "create-props-markdown": "node scripts/props-table-gen/generatePropsTables.js",
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:build": "lerna ls -a | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
```

Comment:
this change will allow lerna ls to pickup private packages, which we want for compiling. we just don't want to release it yet.